src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAOAEPKeyExchangeFormatter.cs

```diff
@@ -47,6 +47,11 @@ public override string Parameters
             get {return null;}
         }
 
+        public RandomNumberGenerator Rng {
+            get { return null; }
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
> But I guess it's easy enough to just blind port the desktop code and be done with it.

Are these types in general not recommended for usage?  If so, an extra field doesn't seem problematic.

Or are these typically created in small numbers?  If so, again an extra field doesn't seem problematic.

But if we expect the field to actually matter, there are ways around it, making this silly property pay-for-play, e.g. using a ```ConditionalWeakTable<RSAOAEPKeyExchangeFormatter, RandomNumberGenerator>```, which will make it much more expensive if the property is actually used, but which will essentially make it free if it's not (other than the extra static field to hold the CWT and a line or two of code in the property).